exercises/word-search/word_search.py

```diff
@@ -1,27 +1,11 @@
 class Point(object):
     def __init__(self, x, y):
-        self.x = x
-        self.y = y
-
-    def __repr__(self):
-        return 'Point({}:{})'.format(self.x, self.y)
-
-    def __add__(self, other):
-        return Point(self.x + other.x, self.y + other.y)
-
-    def __sub__(self, other):
-        return Point(self.x - other.x, self.y - other.y)
-
-    def __eq__(self, other):
```

Comment:
Typically, `__eq__` is implemented for the user to ensure that tests work correctly. Removing the other methods is fine, but please keep `__eq__` and the body of `__init__`.